portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,216 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Rubinstein</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body id = "body">
```

Comment:
Spaces around equals signs in HTML files are uncommon. For consistency with most of the code in this file (and most HTML files elsewhere), please use something like `id="body"` throughout this file.

Reply:
Changed here, and throughout the file in my last push to this branch